index.md

```diff
@@ -30,10 +33,15 @@ please see [the setup instructions]({{ page.root }}{% link setup.md %}).
 > 4.  Put lesson episodes in `_episodes` (or `_episodes_rmd` if you are writing in RMarkdown).
 > 5.  Run `make serve` to preview the lesson website locally.
 > 6.  Do *not* commit the generated HTML files in the `_site` directory.
-> 7.  Style blocks and code samples by putting `{: .stylename}` on a newline *after* the block or code.
+> 7.  Style blocks and code samples by putting `{: .stylename}` on a newline *after* the block or
+      code.
 > 8.  Put solutions inside challenges using nested blockquotes.
 > 9.  File issues and template fixes in the [styles repository][styles],
 >     and enhancements to this documentation in this one.
 {: .checklist}
 
 {% include links.md %}
+
+[curriculum-handbook]: https://carpentries.github.io/curriculum-development/
+[tech-intro]: https://carpentries.github.io/curriculum-development/technological-introductions.html
+
```

Comment:
Currently, the `{% include links.md %}` line must be the last non-empty line in the file (see `check_reference_inclusion` function in `bin/lesson_check.py`.

Reply:
Thank you @maxim-belkin. Pardon my inexperience, but is that documented anywhere? Could be related to carpentries/styles#419.